frontend2/src/components/team/ScrimmageSettings.tsx

```diff
@@ -32,52 +45,55 @@ const ScrimmageSettings: React.FC = () => {
-    // make API calls to update only if necessary
-    if (
-      !isPresent(ranked) ||
-      !isPresent(unranked) ||
-      (team?.profile?.auto_accept_ranked === ranked &&
-        team.profile.auto_accept_unranked === unranked)
-    ) {
-      return;
-    }
+  // useEffect(() => {
+  //   // make API calls to update only if necessary
+  //   if (
+  //     !isPresent(ranked) ||
+  //     !isPresent(unranked) ||
+  //     (team?.profile?.auto_accept_ranked === ranked &&
+  //       team.profile.auto_accept_unranked === unranked)
+  //   ) {
+  //     return;
+  //   }
 
-    let isActive = true;
-    const update = async (): Promise<void> => {
-      let updatedTeam: TeamPrivate;
-      try {
-        updatedTeam = await updateTeamPartial(episodeId, {
-          auto_accept_ranked: ranked,
-          auto_accept_unranked: unranked,
-        });
-      } catch (error) {
-        // TODO: Add a notif here indicating that the update failed
-        console.error(error);
-        return;
-      }
-      const newProfile = team?.profile;
-      if (isActive && isPresent(newProfile) && isPresent(team)) {
-        // only update the ranked / unranked of the team profile to avoid
-        // race conditions with other team profile updaters on this page
-        refreshTeam({
-          ...team,
-          profile: {
-            ...newProfile,
-            auto_accept_unranked: updatedTeam.profile?.auto_accept_unranked,
-            auto_accept_ranked: updatedTeam.profile?.auto_accept_ranked,
-          },
-        });
-      }
-    };
-    void update();
-    return () => {
-      isActive = false;
-    };
-  }, [ranked, unranked, episodeId]);
+  //   let isActive = true;
+  //   const update = async (): Promise<void> => {
+  //     let updatedTeam: TeamPrivate;
+  //     try {
+  //       updatedTeam = await updateTeamPartial(episodeId, {
+  //         auto_accept_ranked: ranked,
+  //         auto_accept_unranked: unranked,
+  //       });
+  //     } catch (error) {
+  //       // TODO: Add a notif here indicating that the update failed
+  //       console.error(error);
+  //       return;
+  //     }
+  //     const newProfile = team?.profile;
+  //     if (isActive && isPresent(newProfile) && isPresent(team)) {
+  //       // only update the ranked / unranked of the team profile to avoid
+  //       // race conditions with other team profile updaters on this page
+  //       refreshTeam({
+  //         ...team,
+  //         profile: {
+  //           ...newProfile,
+  //           auto_accept_unranked: updatedTeam.profile?.auto_accept_unranked,
+  //           auto_accept_ranked: updatedTeam.profile?.auto_accept_ranked,
+  //         },
+  //       });
+  //     }
+  //   };
+  //   void update();
+  //   return () => {
+  //     isActive = false;
+  //   };
+  // }, [ranked, unranked, episodeId]);
 
-  if (
-    teamState !== TeamStateEnum.IN_TEAM ||
+  // TODO: figure out a more elegant way to handle loading than blocking the UI
```

Comment:
The more elegant way would just be to put spaces like this where data is pending
![image](https://github.com/battlecode/galaxy/assets/40174697/dc14af01-8c1b-4b9a-8b97-1db4031d329c)

Reply:
TODO by someone sometime in the future... not now 😃